mne/datasets/utils.py

```diff
@@ -207,11 +206,16 @@ def _data_path(path=None, force_update=False, update_path=True, download=True,
     # import pooch library for handling the dataset downloading
     pooch = _soft_import('pooch', 'dataset downloading', strict=True)
 
+    # extract configuration parameters
+    this_dataset = MNE_DATASETS[name]
+    config_key = this_dataset['config_key']
+    folder_name = this_dataset['folder_name']
```

Comment:
you redefine `this_dataset` below, better to use different var names

```suggestion
    dataset_dict = MNE_DATASETS[name]
    config_key = dataset_dict['config_key']
    folder_name = dataset_dict['folder_name']
```